pandas/tests/io/parser/data/bom_first_line.txt

```diff
@@ -0,0 +1,9 @@
+﻿"Project ID"	"Project Name"	"Product Name"	"Content Type"	"Creation Date"	"Translator"	"Reviewer"	"Lanugage"	"File Name"	"Segment Sequence"	"Word Count"	"TM Score"	"Match Type"	"AT Type"	"MT Engine"	"MT Configuration Name"	"PE Distance"	"Length After Automatic Translation"	"Change in %"	"Source"	"After Automatic Translation"	"Target"
```

Comment:
Do we need an entire file for this or can you just construct a StringIO object directly? Minimal tests would be preferable

Reply:
Yes, I can repeat the error using StringIO.